### PR TITLE
FT: Add 'ReplicationStatus' value to objectHead

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: lib/utilities/collectResponseHeaders.js [hunk 1]
```diff
@@ -65,6 +65,10 @@ function collectResponseHeaders(objectMD, corsHeaders, versioningCfg,
         responseMetaHeaders['x-amz-tagging-count'] =
           Object.keys(objectMD.tags).length;
     }
+    if (objectMD.replicationInfo.status) {
```
Comment (rahulreddy) [inline lib/utilities/collectResponseHeaders.js]: On AWS, do you get replication status when you get object as well?

Comment (bennettbuchanan) [inline lib/utilities/collectResponseHeaders.js]: It is included. 

Example response from AWS:
```
{ AcceptRanges: 'bytes',
  LastModified: 'Wed, 07 Jun 2017 17:41:46 GMT',
  ContentLength: '18',
  ETag: '"c40d3816c5c18ee3ed7a74f8bee0e289"',
  VersionId: 'zrsEAAbJywjnn.roRlVpZc.HWE.OgCRu',
  ContentType: 'binary/octet-stream',
  Metadata: {},
  ReplicationStatus: 'FAILED',
  Body: <Buffer 66 69 6c 65 20 62 6c 61 68 20 63 6f 6e 74 65 6e 74 0a> }
```

Scality S3 response with this PR:
```
{ LastModified: 'Wed, 28 Jun 2017 18:32:09 GMT',
  ContentLength: '18',
  ETag: '"c40d3816c5c18ee3ed7a74f8bee0e289"',
  VersionId: '39383530313332353237303239333939393939395247303031202030',
  WebsiteRedirectLocation: '',
  Metadata: {},
  ReplicationStatus: 'PENDING',
  Body: <Buffer 66 69 6c 65 20 62 6c 61 68 20 63 6f 6e 74 65 6e 74 0a> }
```

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by rahulreddy** [comment]
Please re-target against master. It's extra work for us to port this again.

**Comment by bennettbuchanan** [comment]
@rahulreddy I've rebased to master. We can revisit this once the changes to objectMD are ported from ft/S3-REP.1.

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Pull request merged**